CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Next
+* Button: Component uitbreiden met "mode" props. Dso-rich-content vervangen met richContentTemplate(). Dso-button-row vervangen met buttonRowTemplate(). ([#2278](https://github.com/dso-toolkit/dso-toolkit/issues/2278))
```

Comment:
Onder `### Tasks`

Tekst moet gelijk zijn aan de issue header en de commit message
(`* Templates: Kleine template verbeteringen ([#2278](https://github.com/dso-toolkit/dso-toolkit/issues/2278))* `) 
en van het eerste commit ook
(`#2278 [Tasks] Templates: Kleine template verbeteringen`)

Reply:
👍

Reply:
(En het moet "Task" zijn)

Reply:
Best verwarrend dat daar dan weer onderscheid tussen changelog en commit message in zit.

Reply:
Tsja dat begrijp ik inderdaad ook wel, maar dit waren mijn overwegingen:

* De commit message gaat over één change/issue, het is raar dat daar meervoud zou staan;
* In de CHANGELOG worden de diverse changes gegroepeerd, en daar is meervoud weer logisch.

Daarom dat ik het in de commit message heb over "Type change" en in de CHANGELOG over "groep" of "categorie".